gtsam/slam/slam.i

```diff
@@ -323,6 +323,9 @@ virtual class KarcherMeanFactor : gtsam::NonlinearFactor {
   KarcherMeanFactor(const gtsam::KeyVector& keys);
 };
 
+template <T = {gtsam::Rot3}>
+const T FindKarcherMean<T>(const std::vector<T>& rotations);
```

Comment:
@varunagrawal @ProfFan question for you all -- do we have support for std::vector yet? or do we need to make a new type alias here such as Rot3Vector?

Reply:
No because of MATLAB end

Reply:
huh? I thought that std::vector worked with wrapper??